docs/asciidoc/highlight/CHANGES.md

```diff
@@ -699,7 +699,7 @@ We're going to redesign the old [demo/test suite][demo] into an interactive
 demo web app. If you're confident front-end developer or designer and want to
 help us with it, drop a comment into [the issue][#542] on GitHub.
 
-[test]: https://github.com/isagalaev/highlight.js/tree/master/test
+[test]: https://github.com/isagalaev/highlight.js/tree/main/test
```

Comment:
unrelated repo, but `highlith.js` _has_ switched to `main` themselves so I guess this is fine

Reply:
I checked all the links to ensure they are properly referenced as well